install-dev-dependencies.sh

```diff
@@ -4,7 +4,7 @@
 set -o errexit -o xtrace
 if type apt-get >/dev/null ; then
   sudo apt-get install libxml2-dev libxml2-utils libxslt1-dev python-dev \
-    firefox chromium-browser zip sqlite3 python-pip libcurl4-openssl-dev
+    firefox chromium-browser zip sqlite3 python-pip libcurl4-openssl-dev libssl-dev
```

Comment:
On a Ubuntu system (that I was running in a VM) I had the problem that running install-dev-dependencies.sh ended in a compiler error not finding "crypto.h" (or something similar). Installing libssl-dev solved this problem for me.

But on another machine running Ubuntu I can't reproduce this right now?!